packages/liveblocks-core/src/room.ts

```diff
@@ -480,6 +480,11 @@ export type Room<
    */
   batch<T>(fn: () => T): T;
 
+  /**
+   * Whether or not the room has storage modifications that have not been acknowledged by the server.
+   */
+  hasPendingStorageModifications(): boolean;
```

Comment:
Do we want to document these in our public documentation, too?

Reply:
👌